pkg/minishift/registration/redhat.go

```diff
@@ -67,7 +72,9 @@ func (registrator *RedHatRegistrator) Register(param *RegistrationParameters) er
 			subscriptionCommand := fmt.Sprintf("sudo -E subscription-manager register --auto-attach "+
 				"--username %s "+
 				"--password '%s' ", param.Username, util.EscapeSingleQuote(param.Password))
+			spinnerView.Start()
 			_, err = registrator.SSHCommand(subscriptionCommand)
+			spinnerView.Stop()
```

Comment:
Also, I'd like to add actual timing. If not to the console, then at least to the debug log.